docs/src/modules/spring/pages/views.adoc

```diff
@@ -1 +1,186 @@
+
+[#event-sourced-entity]
+== Creating a View from an Event Sourced Entity
+
+You can create a View from an Event Sourced Entity by using events that the Entity emits to build a state representation. Using a Customer Registry service example, to create a View for querying customers by name:
+
+This example assumes a Customer equal to the previous example and an Event Sourced Entity that uses this Customer and it is in charge of producing the events that update the View. These events are defined as subtypes of the class `CustomerEvent` following standard https://github.com/FasterXML/jackson-annotations#handling-polymorphic-types[Jackson notation] like this: 
+
+.src/main/java/customer/api/CustomerEvent.java
+[source,java]
+----
+include::example$spring-eventsourced-customer-registry/src/main/java/customer/api/CustomerEvent.java[tags=class]
+----
+<1> Includes the logical type name (defined in implementation classes) as type 
+<2> Required for deserialization and to define the logical type name 
+
+Adding `@JsonTypeInfo` is needed so that Jackson can read back correctly the Object type when deserializing. By using `name`, you set its value as the fully qualified message name instead of the using the fully qualified Java class. This is called a `logical type name` and it allows you to refactor the class to another package after you have serialized messages of this type. Otherwise, after such a refactor the objects serialized with the old package won't be recognized as the same type. 
+
+
+=== Define the View to consume events
+
+The definition of the view for an Event Sourced Entity is the same as for a Value Entity. However, in this example, the subscription is at the method level rather than the type level. The advantage of this approach is that you can create multiple methods to handle different events.  
+
+Every time an event is processed by the view, the state of the view can be updated. You can do this with the `.updateState` method, which is available through the `effects()` API. Here you can see how the View is updated with a new name: 
+
+ .src/main/java/customer/view/CustomerByNameView.class
+[source,java]
+----
+include::example$spring-eventsourced-customer-registry/src/main/java/customer/view/CustomerByNameView.java[tags=class]
+----
+<1> updating the state of the view with the new name 
+
+An Event Sourced entity can emit many types of events. You need to define a method for each event type. They return an `UpdateEffect`, which describes next processing actions, such as updating the view state.
+
+NOTE: This type of update transformation is a natural fit for Events emitted by an Event Sourced Entity, but it can also be used for Value Entities. For example, if the View representation is different from the Entity state you might want to transform it before presenting the View to the client.
```

Comment:
The odd thing about the gRPC API is that we need to add `transform_updates: true` when consuming events from an event sourced entity. 

That creates the need to have to add such a paragraph. Makes sense for gRPC, but loses its weight in the Spring SDK. If there is a method, is a transformation, period. 

I think we can remove this and talk about transformation in the part that we explain how to create Views from Value Entities. Transformation simply doesn't make sense for Event Sourced entities because it's always a transformation from events to derived view state.

Reply:
I'm not sure I follow. Let's discuss this.

Reply:
after our call is clear no mention of transformations should appear in the docs